modules/test/playwright/tests/calendar-web/miniCalendar.spec.ts

```diff
@@ -110,3 +125,21 @@ test('ensure that mini calendar accessibility properties are maintained after ch
 		'polite'
 	);
 });
+
+test('ensure that next and previous buttons acessibility label in the mini calendar are translated to portuguese', async ({
+	calendarWidgetPage,
+	page,
+	site,
+}) => {
+	await page.goto(`/pt/web${site.friendlyUrlPath}${layout.friendlyUrlPath}`);
```

Comment:
Can you test whether just by modifying the user's language we can achieve the expected result? If so, we will not need to update the URL or change the language.

This provides more security in knowing that we will not leave any language garbage for other tests.

You can try something like:



```
// Pseudocode

test('test name', async({
    x,
    y,
}) => {
    const user = await creatUser();
    
    apiHelpers.data.push({
		id: user.id,
		type: 'userAccount',
	});
    
    await performUserSwitch(page, user.alternateName);
    
    await page.locator('button[data-qa-id="userPersonalMenu"]').click();

	await page.getByRole('menuitem', {name: 'Account Settings'}).click();

	await accountSettingsPage.selectAccountLanguage('pt_BR');
	
    await page.goto(`calendar page`);
    
    await expect(
		calendarWidgetPage.miniCalendarNextMonthButton
	).toHaveAccessibleName('Ir para o próximo mês');

	await expect(
		calendarWidgetPage.miniCalendarPastMonthButton
	).toHaveAccessibleName('Ir para o mês anterior');
})

```

If the result is not achieved by modifying the user's language but rather that of the site, we can create an isolated site using the fixture. Add the calendar within this site and perform the language change (configuration > site settings > localization) only in the context of this site.